client/src/Components/Pages/PaymentFlow/SelectAmount/SelectAmount.js

```diff
@@ -3,17 +3,23 @@ import React, { useState } from 'react';
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux';
 
-export const SelectAmount = (props) => {
+import { setTip } from './../../../../Actions/TipActions';
+
+const SelectAmount = (props) => {
   const [amount, setAmount] = useState(5)
 
   // Stretch goal: we can set user's currency in database and store it in Redux store
-  const currency = "$"
 
   const submitHandler = () => {
-    // Go to 'Enter Code'
+    props.setTip(amount)
     props.history.push('/tip')
   }
 
+  const currency = props.user.default_currency === 
+    "btc" ? "₿" 
```

Comment:
I'm considering some changes to the API... I could change it to accept the appropriate currency symbol and amount rather than just the currency code maybe.

Reply:
This is a nice solution though 👍

Reply:
I could add a `switch() case` for different currency codes and then returning the appropriate currency symbol as a workaround. Let me know what you plan on doing!